docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -38,10 +38,11 @@ This table links to the release notes for each major release. These release note
 ==============================      =================================================
 Ansible Release                     Status
 ==============================      =================================================
-devel                               In development (2.10 unreleased, trunk)
+devel                               In development (2.11 unreleased, trunk)
+`2.10 Release Notes`_               Maintained (security **and** general bug fixes)
```

Comment:
```suggestion
`2.10 ansible-base Release Notes`_               Maintained (security **and** general bug fixes)
```

Reply:
my guess is the above suggested edit will mess up the table and you'll have to extend the ==== headers above to make it all fit.. cuz yeah, tables in .rst are just that fun :-P